docs/design/design.md

```diff
@@ -135,56 +192,82 @@ Other flags that are set:
  - `--requestheader-client-ca-file` to `front-proxy-ca.crt`
  - `--admission-control` to `NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota`
```

Comment:
this is now updated to something else...
Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota